lib/tdf3/src/client/DecoratedReadableStream.ts

```diff
@@ -33,7 +33,7 @@ export abstract class DecoratedReadableStream {
   upsertResponse?: UpsertResponse;
 
   constructor(byteLimit: number, underlyingSource: UnderlyingSource) {
```

Comment:
If we don't allow byteLimit to be set anymore should we go through and remove it entirely?